src/common/Chart/MlChart.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useRef, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { Chart, registerables } from 'chart.js'
+import Loader from '../Loader/Loader'
+import classnames from 'classnames'
+
+import './mlChart.scss'
+
+Chart.register(...registerables)
+
+const MlChart = ({ config }) => {
+  const canvasRef = useRef()
+  const [isLoading, setIsLoading] = useState(true)
+  const canvasClassNames = classnames(isLoading && 'hidden')
+
+  useEffect(() => {
+    const ctx = canvasRef.current.getContext('2d')
+    const mlChartInstance = new Chart(ctx, {
+      ...config,
+      options: {
+        ...config.options,
+        animation: {
+          ...config.options.animation,
+          onComplete: () => {
+            setIsLoading(false)
+
+            if (config?.options?.animation?.onComplete) {
+              config.options.animation.onComplete()
+            }
```

Comment:
`?.` operator has two siblings: `?.[]` and `?.()`.
`value?.[expr]` is like `value[expr]` but will not throw in case `value` is `null` or `undefined`.
`value?.(params)` is like `value(params)` but will not throw in case `value` is `null` or `undefined`. (It _will_ throw ` TypeError: value is not a function` in case `value` is not a function.)

I think it’s worth trying to do this:

```suggestion
            config?.options?.animation?.onComplete?.()
```

Unless ESLint, Prettier or any other thing shouts…

Reply:
ESLink doesn't allow to do that
![image](https://user-images.githubusercontent.com/25711177/110443868-0ffbb980-80c5-11eb-922c-74a330df5395.png)

Reply:
I figured. Then ignore my comment.